pyproject.toml

```diff
@@ -45,10 +46,7 @@ dynamic = ["version"]
 test = [
   "pytest",
   "matplotlib",
-  "xarray",
-  "pandas",
   "bs4",
-  "numba",
```

Comment:
Can you filter based on python version instead so that these packages are installed with py<3.12?
Or do we actually need test dependencies here? I don't see a real use for them since the listing here is for users.

Reply:
I think it is used by the wheel build, i.e., does not affect normal CI. If I remove it then even fewer tests will run in wheel builds, but maybe that is ok?

Reply:
I think we don't test the wheels anyway in other packages. Not saying that this is good though.
Leave it as is, otherwise, we'd need too many importskip calls.